docset/winserver2022-ps/failoverclusters/Resume-ClusterNode.md

```diff
@@ -96,8 +96,8 @@ Accept wildcard characters: False
 ### -Failback
 
 Sets the policy to bring back drained workloads to the node. The acceptable values for this
-parameter are:NoFailback, Immediate, and Policy. Policy can be configured to not failback, failback
-immediately, or failback only during specific hours.
+parameter are: **NoFailback**, **Immediate**, and **Policy**. Policy can be configured to not
+**failback**, **failback immediately**, or **failback** only during specific hours.
```

Comment:
Specific values for parameters should be formatted as code. I'm not sure the items in the second sentence need formatting, they read to me as explaining what the previous values do.

```suggestion
parameter are: `NoFailback`, `Immediate`, and `Policy`. Policy can be configured to not
failback, failback immediately, or failback only during specific hours.
```

Not necessary for this PR, but it might read better to have each item on a line with the explaination beside it, like:

```suggestion
parameter are:

- `NoFailBack` - Don't failback at all.
- `Immediate` - Failback immediately.
- `Policy` - Failback only during specific hours.
```